src/helpers/issue.ts

```diff
@@ -95,6 +95,22 @@ export const listAllIssuesForRepo = async (state: "open" | "closed" | "all" = "o
   return issuesArr;
 };
 
+export const getIncentivizedUsers = async (issue_number: number) => {
+  const comments = await getAllIssueComments(issue_number);
+  const incentiveComment = comments.filter((comment) => comment.body.startsWith("/comment-incentives"));
+  const parts = incentiveComment[0].body.split("");
+  parts.shift();
+  const users: { enable: boolean; users: string[] } = { enable: false, users: [] };
+  for (const part of parts) {
+    if (part.startsWith("@")) {
+      users.users.push(part.substring(1));
+    } else if (part == "true") {
```

Comment:
```suggestion
    } else if (part === "true") {
```
use `===` whenever possible

also check if part is `false` because it could be anything and we should throw error if it's not `true`, `false` or starts with @.

if `true` or `false` is not provided, then what should be default? @pavlovcik

Reply:
@me505 you need to explicitly check like `if (part === "false")`

Reply:
> if true or false is not provided, then what should be default? @pavlovcik

It should throw an error and ignore the input from the user (i.e. no effect.)